unifold/inference.py

```diff
@@ -26,49 +29,54 @@ def get_device_mem(device):
     else:
         return 40
 
+
 def automatic_chunk_size(seq_len, device, is_bf16):
     total_mem_in_GB = get_device_mem(device)
-    factor = math.sqrt(total_mem_in_GB/40.0*(0.55 * is_bf16 + 0.45))*0.95
-    if seq_len < int(1024*factor):
+    factor = math.sqrt(total_mem_in_GB / 40.0 * (0.55 * is_bf16 + 0.45)) * 0.95
+    if seq_len < int(1024 * factor):
         chunk_size = 256
         block_size = None
-    elif seq_len < int(2048*factor):
+    elif seq_len < int(2048 * factor):
         chunk_size = 128
         block_size = None
-    elif seq_len < int(3072*factor):
+    elif seq_len < int(3072 * factor):
         chunk_size = 64
         block_size = None
-    elif seq_len < int(4096*factor):
+    elif seq_len < int(4096 * factor):
         chunk_size = 32
         block_size = 512
     else:
         chunk_size = 4
         block_size = 256
     return chunk_size, block_size
 
+
 def load_feature_for_one_target(
     config, data_folder, seed=0, is_multimer=False, use_uniprot=False
 ):
     if not is_multimer:
-        uniprot_msa_dir = None
+        uniprot_msa_feature_dir = None
         sequence_ids = ["A"]
         if use_uniprot:
-            uniprot_msa_dir = data_folder
+            uniprot_msa_feature_dir = data_folder
 
     else:
-        uniprot_msa_dir = data_folder
+        uniprot_msa_feature_dir = data_folder
         sequence_ids = open(os.path.join(data_folder, "chains.txt")).readline().split()
-    batch, _ = load_and_process(
+    batch, _ = load_and_process_ss(
```

Comment:
this will harm normal unifold load_and_process. check if the two functions are equivalent, and use old fn name.